README.md

```diff
@@ -136,6 +136,40 @@ To make this project runnable via IntelliJ, do the following:
   is configured according to your needs
 
 
+## How to Run Tests
+
+### Test controls
+
+The project uses the `.sbtrc` [(link)](https://www.scala-sbt.org/1.x/docs/Best-Practices.html#.sbtrc) file to manage and control the execution of integration tests. The Integration tests are the ones with the tag `Integration test.` Below are the commands configured in the `.sbtrc` file to facilitate different testing scenarios:
```

Comment:
tag `IntegrationTest`

Reply:
Fixed in commit - cb82439f4391f2565a18980153f778c9b5fd58cb.